packages/core/src/domain/medical-record-summary.ts

```diff
@@ -16,6 +16,10 @@ export const createMRSummaryFileName = (
   return filePath;
 };
 
+export const createMRSummaryFileNameWithNoExtension = (cxId: string, patientId: string): string => {
+  return createFilePath(cxId, patientId, createFileSuffixBeforeExtension());
```

Comment:
See how we're duplicating the logic to build the MR file name here and on `createMRSummaryFileName`.

I suggest we update `createMRSummaryFileName` to call `createMRSummaryFileNameWithNoExtension` and then add the extension.

This way, `createMRSummaryFileNameWithNoExtension` becomes the source of truth to build the MR file name (path).